### DISCUSSION TITLE
[Unknown Mod][Creature Information window] Information will be centered on the right side of the window, as in the original game, instead of being centered on the colon character

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7013] [Ideas]
In the creature information window, the information will be centered on the right side of the window, as in the original game, instead of being centered on the colon character.
It's nicer presented this way.
This is not optional.

In original game:

![Info text creature centered 001](https://user-images.githubusercontent.com/43583994/232224482-34c643f8-c6a9-463d-8f6f-353621affc27.png)

In fheroes2:

![Info text creature centered 002](https://user-images.githubusercontent.com/43583994/232224499-16294bdb-cb2e-48fb-9141-946913a63295.png)

This can be implemented directly in fheroes2.

**Comment by Mr-Bajs** [comment]
The fheroes2 looks better in my mind. But I do think that many translations struggle to fit the translations into the windows. Though the OG layout would give the translations more freedom and space to fit within the frame.
